src/app/coursegrab/utils/push_notifications.py

```diff
@@ -134,9 +133,7 @@ def send_emails(section, emails):
 
     message = Mail(
         from_email=("mailer@coursegrab.me", "CourseGrab"),
-        subject="Course ID {0}: {1}, {2} is now open!".format(
-            section["catalog_num"], section["title"], section["section"]
-        ),
+        subject=f"Course ID {section['catalog_num']}: {section['title']}, {section['section']} is now open!",
```

Comment:
Should we rewrite the subject to be consistent with the push notification message?

i.e. `subject=f"{serialized_section['subject_code']} {serialized_section['course_num']} {trimmed_section_name} is Now Open",`

Reply:
Yea I was also thinking about that - design hasn't iterated over what the email notifications would look like, but I guess I can make it consistent with push notifications for now!